paper.tex

```diff
@@ -976,27 +978,28 @@ \subsection*{Robustness to partial and incorrect information}
 % 9   py_unary_coal_0.25    200.0   323.054926  154.422588  21.104016
 % 10  py_fully_simied_0.25  200.0   0.000000    117.096174  33.241986
 Given this disparity in the number of polytomies introduced, the 
-effects on the likelihood across the three examples are not entirely
-comparable. For the ARGs containing unary nodes, 
+effects on the likelihood across the three examples are not entirely comparable.
+For the ARGs containing unary nodes, 
 removing 25\% of the internal 
-nodes has negligible effect, and even on the fully simplified 
-ARG has a very minor effect on location of the minimum.
+nodes has negligible effect on the likelihood curve in Fig~\ref{fig:lik-surface},
+and even on the fully simplified 
+ARG has only a very minor effect on location of the minimum.
 
 Fig~\ref{fig:lik-surface} also shows results for ARGs inferred from the 
 simulated data using tsinfer and tsdate~\citep{wohns_unified_2022},
 using default parameters for tsinfer, and the true values of mutation
 rate and $N_e$ for tsdate.
 In the first, we use the default output of tsinfer 
 (2,148 nodes; 5,589 edges; 1,307 trees; 3,039 mutations).
-This ARG has an average of 192 unary nodes, 120 binary nodes, and 32 nodes
```

Comment:
The average is correct here, I don't think we should remove it?

Reply:
oh sorry I should have asked about this one - I didn't understand what the average was across?

Reply:
The trees along the sequence

Reply:
oh that is in fact what the sentence says, lol, sorry